package.json

```
@@ -10,9 +10,13 @@
     "sourcecred": "0.7.0-beta-4"
   },
   "scripts": {
+    "clean": "rimraf cache output site",
```

```suggestion
    "clean": "rimraf cache",
```

Let's only remove the cache, so as to match the description in the README. In general users often want to remove cache, and removing the site or output may surprise the user. Removing the cache has no effect on the behavior other than forcing usage of fresh data, and updating info from cache, and it may take a little longer, but it's always "safe" to do it in a workflow.
Removing the site would leave the user with broken symlinks (site command makes simlinks as well as the site directory).
The output directory contains the graphs for each plugin, and if the user doesn't have credentials for every plugin, it's possible that the user will not be able to regenerate some of that data.

---

Based on #4, we will probably get rid of the symlinks altogether and just deploy the site to a separate branch. The only reason we need the site folder here is for users running / debugging locally. I'm actually inclined to add the site folder to gitignore as well since its "generated code".

The reason I added the output folder is because I would get errors when running the commands after changing the config (e.g. changing the discord guildId). A smarter way to do it would be for plugins to handle invalidating the output so we know when to delete it when running `sourcecred graph`.

Most users who fork this to use it for the first time will be running into errors pretty much right away unless they delete the existing output directory. Maybe we can add a separate `clean:all` command that deletes the output as well?